functions/get_prices/test_get_prices.py

```
@@ -0,0 +1,138 @@
+        mock_official_collection = Mock()
+        mock_official_collection.stream.return_value = [mock_official_doc]
+        
+        def collection_side_effect(name):
+            if name == 'kaitori_prices':
+                return mock_kaitori_query
+            elif name == 'official_prices':
+                return mock_official_collection
+        
+        mock_db.collection.side_effect = collection_side_effect
+        
+        response_body, status_code, headers = get_prices(request)
```

Ruff (0.14.2)

Unpacked variable `headers` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)